crates/transaction-pool/src/config.rs

```diff
@@ -30,7 +30,28 @@ pub struct PoolConfig {
     pub max_account_slots: usize,
     /// Price bump (in %) for the transaction pool underpriced check.
     pub price_bumps: PriceBumpConfig,
+    /// Flag to disable local transaction exemptions.
+    pub tx_pool_policy: TxPoolPolicy,
 }
+/// `TxPoolPolicy` holds configuration settings that dictate the behavior of the transaction pool.
+
+#[derive(Debug, Clone, Default)]
+pub struct TxPoolPolicy {
+    /// If set to `true`, local transactions (transactions originating from the local node) will
+    /// not be exempt from certain checks and limitations that are applied to non-local
+    /// transactions.
+    pub no_locals: bool,
+}
```

Comment:
I don't think we need an additional container type for this

Reply:
> I don't think we need an additional container type for this

no problem, since i saw that PoolConfig got lot of container type for it field, i tought it could be good to make it same for no_locals :)